src/theory/sets/solver_state.h

```diff
@@ -200,10 +203,12 @@ class SolverState
   /** debug print set */
   void debugPrintSet(Node s, const char* c) const;
 
- private:
   /** constants */
   Node d_true;
   Node d_false;
+
+private:
```

Comment:
Not a huge fan of making data members public. Maybe define getters?

Reply:
Done.  I wonder if there is a nice way to globally access these 2 constants which are common. @ajreynol  pointed out before using static objects to hold these constants afftects the performance.

Reply:
Yeah, we should probably add them to the `NodeManager` (in another PR). This shouldn't negatively affect performance but we should double check.